src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPrepearedPeople } from '../../utils/getPrepearedPeople';
+
+export const PeoplePage = () => {
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Better to have `false` by default

Reply:
If I use `false` by default, for a moment appears not loader but "There are no people on the server" text

Reply:
@Koliras so that there is no such behavior, it is necessary to set the status to `true` before starting to receive data from the server.